tests/suites/test_suite_ssl.function

```diff
@@ -5589,4 +5594,88 @@ void cid_sanity( )
 }
 /* END_CASE */
 
+/* BEGIN_CASE depends_on:MBEDTLS_X509_CRT_PARSE_C:MBEDTLS_USE_PSA_CRYPTO:MBEDTLS_PKCS1_V15:MBEDTLS_SSL_PROTO_TLS1_2:MBEDTLS_RSA_C:MBEDTLS_ECP_DP_SECP256R1_ENABLED:MBEDTLS_SSL_PROTO_DTLS:MBEDTLS_ENTROPY_C:MBEDTLS_RSA_C:MBEDTLS_ECP_DP_SECP384R1_ENABLED:MBEDTLS_CTR_DRBG_C:MBEDTLS_ECP_C */
+void raw_key_agreement_fail( )
```

Comment:
This test function is fairly simple, but still complex enough that I'd like to be sure that changing the server's public key is responsible for the failure of the handshake. So I'd like to run this test function twice: once for a good case, and once for the desired failure case.